en/flying/led_meanings.md

```diff
@@ -4,8 +4,13 @@
 
 ![LED meanings](../../images/led_meanings.gif)
 
-> **Note** **GPS Lock is required to takeoff: ** 
-  The LED will only turn green (ready to execute guided mission) if PX4 is able to validate the global position. If the LED does not change from blue to green, verify that the GPS module is properly attached, Pixhawk is reading your GPS properly, and that the GPS is sending a proper GPS position.
+> **Note** **A valid global position estimate is required to takeoff:** 
```

Comment:
@priseborough 
1. Do the LEDs reflect GPS or position estimator status? If it is the former, is it therefore possible to have the light effectively saying "ready to arm" because you have GPS lock but the vehicle is not actually ready to take off because the estimator isn't reporting sufficient quality?

Reply:
That is correct. The GPS can have a 3D lock, but the EKF will not start using it ad report a valid global position until the output quality from the GPS has passed the EKF's internal checks. The estimator_status.gps_check_fail_flags message in the logs shows which GPS quality checks are failing

Reply:
Thanks - I have added the fail case to LED meanings doc and the gps_check_fail_flags message info to the Preflight checks doc.